_commands/basics/rm.md

```diff
@@ -0,0 +1,55 @@
+---
+---
+
+rm
+--
+
+`rm` is used to delete one or multiple files from your current directory. 
+
+~~~ bash
+$ rm file1.txt file2.txt
+
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `rm -d`
```

Comment:
`-r` and `-R` (identical) both imply `-d`, from the OS X man page:

```
     -d          Attempt to remove directories as well as other types of
                 files.
     -R          Attempt to remove the file hierarchy rooted in each file
                 argument.  The -R option implies the -d option.  If the -i
                 option is specified, the user is prompted for confirmation
                 before each directory's contents are processed (as well as
                 before the attempt is made to remove the directory).  If
                 the user does not respond affirmatively, the file hierar-
                 chy rooted in that directory is skipped.

     -r          Equivalent to -R.
```

My habit is `-rf` as well, but not sure where that was learned or why to be honest.